middleware/logger/templateChain.go

```diff
@@ -0,0 +1,67 @@
+package logger
```

Comment:
Consistency issues, it is recommended to use snake case style，like template_chain.go